### ISSUE TITLE
[maven] jreleaser-maven-plugin-1.3.0 fails execution

### ISSUE THREAD

**Issue by aalmiray** [jreleaser/jreleaser#1007]
### Task List

- [x] Steps to reproduce provided
- [x] Stacktrace (if present) provided
- [ ] Example that reproduces the problem (link to git repository is ideal)
- [x] Full description of the issue provided (see below)

### Steps to Reproduce

1. Configure `jreleaser-maven-plugin:1.3.0` in a Maven build.
2. Attempt a release

### Expected Behaviour

Successful release is posted.

### Actual Behaviour

Build failure. More details at https://github.com/jreleaser/jreleaser/discussions/1006

### Environment Information

- **Operating System**: N/A
- **JReleaser Version:** 1.3.0
- **JDK Version:** N/A

**Comment by aalmiray** [comment]
Current workaround is to update the `commons-io` version used by the plugin like so

```xml
<plugin>
  <groupId>org.jreleaser</groupId>
  <artifactId>jreleaser-maven-plugin</artifactId>
  <version>1.3.0</version>
  <dependencies>
    <dependency>
      <groupId>commons-io</groupId>
      <artifactId>commons-io</artifactId>
      <version>2.11.0</version>
    </dependency>
  </dependencies>
</plugin>
```

**Comment by dsgrieve** [comment]
Confirming that this workaround is effective.

**Comment by aalmiray** [comment]
Unless a critical bug is found that would prompt an immediate release (such as `1.3.1`) this fix remains for `1.4.0` currently scheduled for Dec 30th 2022. 

We do not publish snapshot releases for plugin artifacts, thus the workaround must remain in place til the next stable release is posted.

**Comment by aalmiray** [comment]
Released in v1.3.1 -> https://github.com/jreleaser/jreleaser/releases/tag/v1.3.1
